app/models/person.rb

```diff
@@ -86,7 +84,7 @@ def target_legislators(json: false, count: Ivr::Call::MAXIMUM_CONNECTIONS)
   private
 
   def update_nation_builder
-    relevant_fields = changed & ["email" , "phone"]
+    relevant_fields = changed & ['email' , 'phone']
```

Comment:
Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.